iceoryx_posh/include/iceoryx_posh/mepoo/segment_config.hpp

```diff
@@ -42,8 +41,8 @@ struct SegmentConfig
         {
         }
 
-        std::string m_readerGroup;
-        std::string m_writerGroup;
+        posix::PosixGroup::string_t m_readerGroup;
```

Comment:
We really need a convention for string aliases... this is the third different format !